AWSS3Tests/AWSS3TransferUtilityTests.swift

```diff
@@ -86,12 +88,22 @@ class AWSS3TransferUtilityTests: XCTestCase {
             dataString += dataString
         }
         testData = dataString.data(using: String.Encoding.utf8)!
+        
+        let timeInterval = Date.timeIntervalSinceReferenceDate
+        generalTestBucket = "ios-v2-s3.periods\(timeInterval)"
```

Comment:
It's probably safe to make the bucket name more generic than "ios-v2-s3.periods", unless each created bucket is being created with periods, or unless the existence of periods is being asserted somewhere (which probably belongs in its own test then). Consider something like "ios-integ-tests-\(timeinterval)". Also, consider converting the timeInterval to milliseconds and truncating to an integer, to ensure the format is consistent and doesn't include trailing decimals (TimeInterval is a Double)

Reply:
Renamed to s3-integ-test-, s3-integ-transfermanager-test-, s3-integ-transferutil-test- so that we can distinguish between tests.